code/modules/mob/living/status_procs.dm

```diff
@@ -69,7 +69,7 @@
 			Your character is unable to move, and drops stuff in their hands. They keep standing, though.
 	* Stuttering			*
 			Your character stutters parts of their messages.
-	*	Weakened				*
+	*	knockdown				*
```

Comment:
You should probably mark these status effects with something that indicates whether they operate in life ticks or in deciseconds
(This is the documentation area for the status effects)

Reply:
Right that's a really good idea